docs/pages/posts/2022/8/27/secureum-a-maze-x-stanford-ctf.mdx

```diff
@@ -426,12 +432,11 @@ This challenge was a lesson on why to never trust DEX's current prices as a pric
 ---
 
 import { Card, Cards } from 'nextra/components'
-import { FileIcon, NewsletterIcon } from '@components/icons'
+import { NewsletterIcon } from '@components/icons'
 
-<Cards>
   <Card
-    icon={<FileIcon />}
+    icon={<NewsletterIcon />}
     title="Secureum A-MAZE-X CTF 2023 At DeFi Security Summit →"
-    href="/posts/2023/7/16/secureum-a-maze-x-ctf-2023-at-defi-security-summit/"
+    href="/posts/2023/7/16/secureum-a-maze-x-ctf-2023-at-defi-security-summit"
   />
```

Comment:
Is missing the discussed backward / forward links to races in addition to the link to the next A-MAZE